include/deal.II/base/geometry_info.h

```diff
@@ -4977,7 +4977,7 @@ inline Tensor<1, dim>
 GeometryInfo<dim>::d_linear_shape_function_gradient(const Point<dim> &,
                                                     const unsigned int)
 {
-  Assert(false, ExcNotImplemented());
+  DEAL_II_NOT_IMPLEMENTED()
```

Comment:
What about we change the syntax to `DEAL_II_NOT_IMPLEMENTED();` (with a semicolon)?